ora2pg/data/ddl_18_2_prod.sql

```diff
@@ -0,0 +1,4162 @@
+-- Generated by Ora2Pg, the Oracle database Schema converter, version 18.2
```

Comment:
Why is this called ddl_18_2_prod? Should this have a more relatable name like oats_schemadump_ddl.sql or something?